roles/foreman_installer/tasks/installer_pr.yml

```
@@ -0,0 +1,17 @@
+---
+- name: "install git"
+  package:
+    name: 'git'
+    state: 'present'
+  tags:
+    - packages
+
+- name: "fetch git PR"
+  shell: >
+      git init &&
+      git remote add {{ installer_fork }}  https://github.com/{{ installer_fork }}/foreman-installer.git &&
+      git fetch &&
+      git branch {{ installer_branch }} origin/{{ installer_branch }} &&
+      git checkout -f {{ installer_branch }}
```

```suggestion
      git checkout --force --branch {{ installer_branch }} origin/{{ installer_branch }}
```

technically, there is no need for `--branch …` as we can also directly checkout the remote branch and don't care that we don't have a local one in this case